src/components/mote/components/innkalling/DialogmoteMoteStatusPanel.tsx

```diff
@@ -60,23 +71,30 @@ const ParticipantInfo = ({
   dialogmote,
   bruker,
 }: ParticipantProps): ReactElement => {
+  const arbeidstakerHarLestInnkallingen = isLestInnkalling(
+    dialogmote.arbeidstaker.varselList
+  );
+  const arbeidsgiverHarLestInnkallingen = isLestInnkalling(
+    dialogmote.arbeidsgiver.varselList
+  );
+
   return (
     <FlexColumn>
       <InfoRow
-        icon={getIcon(true)}
+        icon={getHarLestIcon(arbeidsgiverHarLestInnkallingen)}
         title={
           <span>
             <b>{texts.naermesteLeder}</b> {dialogmote.arbeidsgiver.lederNavn},
-            avklar hva som skal vises her
+            {getHarLestText(arbeidsgiverHarLestInnkallingen)}
```

Comment:
Vi har jo datoen for når det ble lest - hadde det vært bra å vise den også?

Reply:
Ja, hørte med Espen om det, men han ville scope det ut nå ettersom visningen her skal endre seg